packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -1526,6 +1541,25 @@ class MemoryConfigurationsDialog extends StatelessWidget {
                 ),
               ],
             ),
+            const SizedBox(height: defaultSpacing,),
+            ...dialogSubHeader(theme, 'General'),
+            Column(
+              children: [
+                Row(
+                  children: [
+                    NotifierCheckbox(
+                        notifier: controller.advancedSettingsEnabled,),
```

Comment:
dartformat needs to be run. Then the line will be:

```
                    NotifierCheckbox(
                      notifier: controller.advancedSettingsEnabled,
                    ),
```

Reply:
Sorry, I forgot to do that